### PR TITLE
249140: Internal Load balancer subnet

### PR DESCRIPTION
**Pull request by ppotturi**
#Description

Added annotation to set ILB subnet and renamed flux config namespace to "flux-config-cluster".

### CODE REVIEW

**Review by DaRoszko — APPROVED** [review]

**Review by ajaysoni-cap — APPROVED** [review]

File: infra/snd/01/nginx-ingress-values.yaml [hunk 1]
```diff
@@ -2,13 +2,16 @@ apiVersion: helm.toolkit.fluxcd.io/v2beta1
...
+  namespace: flux-config
 spec:
   chart:
     spec:
       version: 0.18.1
   values:
     controller:
+      service:
+        annotations:  
+          service.beta.kubernetes.io/azure-load-balancer-internal-subnet: "SNDADPNETSU1401" #TODO: Need to update once the platform values are available.
```
Comment (aadev1) [inline infra/snd/01/nginx-ingress-values.yaml:14]: @ppotturi - Are we leaving SNDADPNETSU1401 hard-coded for now?  We could use the postbuild substitution in the Cluster Bicep template, similar to how we've done it for the ASO secret values and App Config values.

Comment (ppotturi) [reply infra/snd/01/nginx-ingress-values.yaml:14]: It will be changed to platform variables once they are available.

Comment (aadev1) [reply infra/snd/01/nginx-ingress-values.yaml:14]: okay cool.

**Review by aadev1 — APPROVED** [review]

File: services/base/namespace.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: config-services-flux
\ No newline at end of file
+  name: flux-config-services
```
Comment (kbabigumira) [inline services/base/namespace.yaml:4]: This can be removed. we don't need this namespace.

Comment (ppotturi) [reply services/base/namespace.yaml:4]: Ajay removed that in his changes.

**Pull request merged**